utils/release_helper.py

```diff
@@ -0,0 +1,788 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+
+"""
+Script created to help maintainers during the release process by automating Github tasks.
+"""
+
+# References:
+# - https://developer.github.com/v3/libraries/
+# - https://github.com/PyGithub/PyGithub
+# - https://pygithub.readthedocs.io/en/latest/github_objects.html
+# - https://docs.github.com/en/rest
+
+from datetime import datetime, timedelta
+from github import Github
+import argparse
+import configparser
+import os.path
+import re
+import subprocess
+
+
+def get_parameter_from_ini(config_file, section, parameter) -> str:
+    config = configparser.ConfigParser()
+    try:
+        config.read(config_file)
+        return config[section][parameter]
+    except Exception as e:
+        print(f'{e} entry was not found - Information cannot be retrieved!')
+        exit(1)
```

Comment:
This is not a big deal, but I would handle the exception by exit only on the top level. IOW, I would either not handle the exception on this level, or handle it by formulating the message, re-raising it, and then do the final handling on the top level.

Reply:
Improved. Thanks for the feedback on this.